### DISCUSSION TITLE
https://zongzi531.com/2020/12/16/img%E6%A0%87%E7%AD%BE%E4%BD%BF%E7%94%A8svg%E5%90%8E%E5%A6%82%E4%BD%95%E6%94%B9%E5%8F%98%E9%A2%9C%E8%89%B2/

### DISCUSSION THREAD

**Discussion by giscus** [zongzi531/zongzi531.github.io#29] [General]
# https://zongzi531.com/2020/12/16/img%E6%A0%87%E7%AD%BE%E4%BD%BF%E7%94%A8svg%E5%90%8E%E5%A6%82%E4%BD%95%E6%94%B9%E5%8F%98%E9%A2%9C%E8%89%B2/

一次知难而上的尝试

https://zongzi531.com/2020/12/16/img%E6%A0%87%E7%AD%BE%E4%BD%BF%E7%94%A8svg%E5%90%8E%E5%A6%82%E4%BD%95%E6%94%B9%E5%8F%98%E9%A2%9C%E8%89%B2/

**Comment by Yungzzhh** [comment]
这字体要看瞎了

**Reply by zongzi531** [reply]
不好意思，影响到你的阅读。我排查了一下是因为自动升级到 hexo7 引起的问题，我已修复次此情况。祝您阅读愉快。

**Comment by rebron1900** [comment]
方案3好用，已经实装，谢谢。

**Reply by rebron1900** [reply]
不过最近有有人反馈，在苹果上好像不兼容，图片直接消失了。

**Reply by zongzi531** [reply]
@rebron1900 方便提供设备信息以及具体的场景吗

**Reply by rebron1900** [reply]
一个网友反馈的，我现在换成直接用svg标签了，hhh

**Reply by zongzi531** [reply]
@rebron1900 有问题欢迎交流
